src/Symfony/Component/Testing/bootstrap.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Component\Testing\DeprecationErrorHandler;
+
+if (class_exists('PHPUnit_Util_ErrorHandler')) {
+    return;
+}
+
+if (PHP_VERSION_ID >= 50400 && gc_enabled()) {
+    // Disabling Zend Garbage Collection to prevent segfaults with PHP5.4+
+    // https://bugs.php.net/bug.php?id=53976
+    gc_disable();
+}
+
+DeprecationErrorHandler::register();
+AnnotationRegistry::registerLoader('class_exists');
```

Comment:
what if Doctrine Annotations is not used in the project ? There is no guarantee given that you don't define it as a dependency. This would trigger a fatal error when loading the composer autoloader currently